test/test_transforms_v2_refactored.py

```diff
@@ -376,35 +376,6 @@ def transform(bbox):
     return torch.stack([transform(b) for b in bounding_boxes.reshape(-1, 4).unbind()]).reshape(bounding_boxes.shape)
 
 
-@pytest.mark.parametrize(
-    ("dispatcher", "registered_input_types"),
-    [(dispatcher, set(registry.keys())) for dispatcher, registry in _KERNEL_REGISTRY.items()],
-)
-def test_exhaustive_kernel_registration(dispatcher, registered_input_types):
```

Comment:
We no longer have an exhaustive registration and thus this test needs to be removed.